build.gradle

```diff
@@ -52,3 +53,7 @@ android {
         abortOnError false
     }
 }
+
+dependencies {
+    compileOnly  'com.android.support:support-annotations:27.0.2'
```

Comment:
I still don't get what exactly we get from this.
Is it an Android-Studio thing?

Reply:
As mentioned before in a comment to #716: it's just annotations part of it for making code look cleaner (e.g. @NonNull or @Nullable etc. JSR-305). It is not specific to Android Studio and is used during compilation.